tests/strategies/test_bagua_strategy.py

```diff
@@ -0,0 +1,40 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import torch
+
+from pytorch_lightning import Trainer
+from pytorch_lightning.strategies import BaguaStrategy
+from tests.helpers.boring_model import BoringModel
+from tests.helpers.runif import RunIf
+
+
+class TestModel(BoringModel):
+    def __init__(self):
+        super().__init__()
+        self.layer = torch.nn.Linear(32, 2)
+
+
+@RunIf(skip_windows=True, bagua=True, min_gpus=2, standalone=True)
```

Comment:
is it not supported on windows?

Reply:
Yes, only on Linux now, and not support CPU as well.

Reply:
can you add an error for it? if the platform is windows and someone tries to use `BaguaStrategy`... or does it fail during bagua installation itself?

Reply:
It will fail on bagua installation.